src/content/docs/tutorials/crates-tui/errors.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Errors
+---
+
+Now that we have a `tui` module, in addition to restoring the state of the terminal at the end of
+`main`, we want to make sure that even when the application panics, we restore the state of the
+terminal back to a normal working state. We also want to print the error to the terminal to that the
+user can see what went wrong.
+
+Rust has a built-in function to set a panic hook:
+
+```rust
+    std::panic::set_hook(Box::new(move |panic_info| {
+        if let Err(err) = crate::tui::restore_backend() {
+            log::error!("Unable to restore terminal: {err:?}");
+        }
+        // TODO: print error to terminal here
+    }));
+```
```

Comment:
Description and example don't align well.

Reply:
I've reworked this now slightly.